pkg/cli/cli_handle.go

```diff
@@ -0,0 +1,147 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/abcxyz/access-on-demand/apis/v1alpha1"
+	"github.com/abcxyz/access-on-demand/pkg/handler"
+	"github.com/abcxyz/access-on-demand/pkg/requestutil"
+	"github.com/abcxyz/pkg/cli"
+	"github.com/posener/complete/v2/predict"
+)
+
+var _ cli.Command = (*CLIHandleCommand)(nil)
+
+// CLIHandleCommand handles CLI requests.
+type CLIHandleCommand struct {
+	cli.BaseCommand
+
+	flagPath string
+
+	flagDebug bool
+
+	// Run Cleanup instead of Do if true.
+	Cleanup bool
```

Comment:
Sorry, I don't think this is the correct architecture. We should have a "do" command and a "cleanup" command. Branching logic based off of a boolean feels like a design smell.

Reply:
In this case, what about make it a flag. so the commands are like do: `aod cli handle -path <path/to/file.yaml>`, cleanup: `aod cli handle -path <path/to/file.yaml> -cleanup`

Reply:
We discussed similar question in https://github.com/abcxyz/access-on-demand/pull/52#discussion_r1240232927. Maybe it is still worthwhile to have a  `wrapper layer, "do": &CLIDoCommand{CLIHandleCommand{dofunc(ctx, *v1alpha1.CLIRequest) error}}, "cleanup": &CLICleanupCommand{CLIHandleCommand{cleanupfunc(ctx, *v1alpha1.CLIRequest) error}}` now?

Reply:
I think this might be better in fact: https://github.com/abcxyz/access-on-demand/pull/55, thank you both! Let me know what you think.